bleak/__init__.py

```diff
@@ -148,12 +149,12 @@ def __init__(
             **kwargs,
         )
 
-    async def __aenter__(self):
-        await self._backend.start()
+    async def __aenter__(self) -> BleakScanner:
+        await self.start()
         return self
 
-    async def __aexit__(self, exc_type, exc_val, exc_tb):
-        await self._backend.stop()
+    async def __aexit__(self, exc_type, exc_val, exc_tb) -> None:
+        await self.stop()
```

Comment:
Change remove. Should I open a PR or do you have any pending changes/commits that this could fit into?

Reply:
Yes, feel free to make a separate PR for it. Always happy to keep improving typing. Just doing small fixes like this as we notice them is easier to review rather than trying to tackle a whole bunch at one time.